protocol/js/flow-types-chat.js

```diff
@@ -16,6 +16,60 @@ export type RPCError = {
   code: number,
   desc: string
 }
+export type WaitingHandlerType = (waiting: boolean, method: string, sessionID: number) => void
+
+// $FlowIssue we're calling an internal method on engine that's there just for us
+const engineRpcOutgoing = (...args) => engine._rpcOutgoing(...args)
```

Comment:
I didn't mean we were building multiple engines. I meant that importing a type has a side effect of building an engine (assuming it didn't exist already).